MojioSDK/mojiosdksrc/src/beta/java/io/moj/mobile/android/sdk/enums/Endpoint.java

```diff
@@ -1,64 +0,0 @@
-package io.moj.mobile.android.sdk.enums;
-
-import android.content.Context;
-import android.text.TextUtils;
-import android.util.Log;
-
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Locale;
-import java.util.MissingResourceException;
-import java.util.Set;
-
-import io.moj.mobile.android.sdk.R;
-
-/**
- * An enumeration of different backend endpoints for the Mojio API.
- * Created by skidson on 15-09-21.
- */
-public enum Endpoint {
-
-    NA("api.moj.io", true, R.array.iso3166_alpha3_na, R.array.iso3166_alpha2_na, R.array.iso639_1_na),
```

Comment:
Since we're no longer using these ISO codes we could delete them in strings.xml too